mlir-sycl/lib/Conversion/SYCLToLLVM/SYCLToLLVM.cpp

```diff
@@ -369,6 +369,42 @@ class ConstructorPattern final
   }
 };
 
+//===----------------------------------------------------------------------===//
+// MethodPattern - Converts SYCLMethodOpInterface to LLVM.
+//===----------------------------------------------------------------------===//
+
+static LogicalResult convertMethod(SYCLMethodOpInterface op,
```

Comment:
do you think is better to put this function as a private member function of `MethodPattern`?